packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -58,6 +68,11 @@ export interface ChangeConnectionState {
 	connected: boolean;
 }
 
+export interface AddClient {
+	type: "addClient";
+	addedChannelId: string;
```

Comment:
it does get used as the id for the dds--see L331. We don't bother setting nice ids elsewhere.

Reply:
i'm confused by L309, why does every new client get a new channel? or am i misunderstanding that.

Reply:
ahh, I see what you mean. There aren't actually new channels created since the mocks hook things up to the same channel and basically ignore that id. But switching the name here to clientId/addedClientId and setting it on a higher level channel should better line up with how things work on a real document and not require much code change in uses. i'll do that.

Reply:
I pushed a change which does that. One way in which it's awkward--the "assert consistency" validation currently only has access to the DDS and it's convenient to throw errors like "client A differs from client B at position 5". This was implemented before using `channel.id`, but I don't think it's doable now. I could just make the consistency validation function take in the two clients, but I kind of like the emphasis that it's meant to assert the two DDS contents match. If you think that should go as well (and we should make all the channelId's match as they would in a real document) I'm ok making that change here.